examples/ios/QuickStartSwiftUI/QuickStart.swift

```diff
@@ -59,121 +61,96 @@ final class Group: Object, ObjectKeyIdentifiable {
-            DispatchQueue.main.async {
-                self.isLoggingInOrOut = false
-                guard error == nil else {
-                    print("Failed to log in: \(error!)")
-                    self.error = error!
-                    return
-                }
-                print("Logged in as user: \(user!.id)")
-                self.user = user
+    init() {
+        // if we have a current user, check if a Realm already exists
+        // for the given configuration. if not, asynchronously open
+        // one. else, assign the existing one
+        if let user = app.currentUser {
+            let configuration = user.configuration(partitionValue: user.id)
+            if Realm.fileExists(for: configuration) {
+                self.realm = try! Realm(configuration: configuration)
+            } else {
+                shouldIndicateActivity = true
+                openRealm(user: user)
             }
         }
-    }
 
-    /// Logs out the current user, if any.
-    func logOut() {
-        self.isLoggingInOrOut = true
-        user?.logOut() { (error) in
-            DispatchQueue.main.async {
-                self.isLoggingInOrOut = false
-                guard error == nil else {
-                    self.error = error
-                    self.user = app.currentUser
-                    return
-                }
-                self.error = nil
-                self.user = nil
+        // monitor log in state and open a realm on login
+        loginPublisher.receive(on: DispatchQueue.main).sink(
+            receiveCompletion: { _ in },
+            receiveValue: { user in
+                self.shouldIndicateActivity = true
+                self.openRealm(user: user)
             }
-        }
+        ).store(in: &cancellables)
+
+        // monitor log out state and unset the realm on log out
+        logoutPublisher.receive(on: DispatchQueue.main).sink(
+            receiveCompletion: { _ in },
+            receiveValue: { user in
+                self.realm = nil
+            }
+        ).store(in: &cancellables)
     }
-}
 
-/// Opens a realm when a user is logged in and retrieves the first group in the 
-/// realm or creates one if no group exists. 
-final class GroupOwner: ObservableObject {
-    @Published var error: Error?
-    @Published var group: Group?
-
-    private(set) var auth: Auth
-
-    private var cancellables = Set<AnyCancellable>()
-
-    init(auth: Auth) {
-        self.auth = auth
-        
-        // Use Combine to watch for changes to the authentication state, i.e.
-        // when someone logs in or out.
-        auth.$user.sink { (user) in
-            if (user == nil) {
-                print("User removed.")
-                self.error = nil
-                self.group = nil
+    /// Open a Realm for a given User.
+    private func openRealm(user: User) {
+        Realm.asyncOpen(configuration: user.configuration(partitionValue: user.id)) { result in
```

Comment:
You changed the functionality here -- the idea is to set people up to be able to add sync later without actually requiring it right now (so, use asyncOpen without actually passing the user configuration)

Reply:
Ah, that would be simple to add back in, I believe that was just a one liner?